.github/workflows/maven.yml

```diff
@@ -10,7 +10,7 @@ on:
     branches: [ main, 1.x ]
 
 env:
-  mvn_options: --batch-mode -Dstyle.color=always -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss.SSS
+  mvn_options: --batch-mode -Dnet.bytebuddy.experimental=true -Dstyle.color=always -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss.SSS
```

Comment:
no need fo `=true`
this should be enough
`-Dnet.bytebuddy.experimental`

Reply:
I just stole it from here: https://github.com/raphw/byte-buddy/issues/1396

Reply:
+1

Reply:
Seems this Windows Build doesn't want to start or finish. I'll kill it

Reply:
👍

I re-ran failed (which includes cancelled). Fingers crossed it actually runs this time.

Reply:
haha, thanks @kingthorin !

Reply:
Hmm, it seems the Windows version doesn't like the build for some reason. Anyone suggestions?

Reply:
I can look into it after work.

Reply:
I cancelled the build, triggered it again, but seems no luck.

Reply:
let's wait till the end to see exception
i guess there should be timeout something about an hour

Reply:
seems I found the reason

1. `mockito` (5.5.0)  depends on `byte-buddy` 1.14.6
2. `mvn dependency:tree` shows that it uses `byte-buddy` 1.12.11
3. it comes from `assertj-core`

As a result exclusion of `byte-buddy`  from `assertj-core` solves the issue and makes it using `byte-buddy` from `mockito`
as a bonus after that there is no need for `-Dnet.bytebuddy.experimental` at all

Reply:
Nice! Thanks for tackling that.